CHANGELOG.md

```diff
@@ -1,5 +1,7 @@
 # v0.12.0: better pickle support, various fixes and better aggregators
 
+* substring filters have been renamed from "substr" to "sub" and "notSubstr" to "notSub" everywhere, for consistency.
+  change is backwards compatible, the old "substr" parameters are still accepted in all places where they've been accepted before.
```

Comment:
this change was after 0.12.0
normally we would have a "master / unreleased" version on top with the changes since last release.
once we do the release, we convert that into the proper version name

Reply:
Ok, I updated it again, how about now?